Framework/DataHandling/inc/MantidDataHandling/LoadEventNexus.h

```diff
@@ -7,14 +7,21 @@
 #include "MantidAPI/IFileLoader.h"
 #include "MantidDataHandling/BankPulseTimes.h"
 #include "MantidDataObjects/EventWorkspace.h"
-#include <nexus/NeXusFile.hpp>
-#include <nexus/NeXusException.hpp>
 #include "MantidDataObjects/Events.h"
 #include "MantidAPI/WorkspaceGroup.h"
 #include "MantidGeometry/Instrument.h"
 #include "MantidGeometry/Instrument/ParameterMap.h"
 #include "MantidKernel/TimeSeriesProperty.h"
 #include "MantidDataHandling/EventWorkspaceCollection.h"
+
+#ifdef _WIN32 // fixing windows issue causing conflict between
+// winnt char and nexus char
+#undef CHAR
```

Comment:
I tried this -- does not work or have not find a way to make it working. 

The conflict occurs when ::Nexus templates are instantiated within ProcessBankData.c where winnt. and nexus.hpp definitions are clashing. 

Naturally I can not predefine only ::Nexus::file as more nexus objects are defined in this header, requering full definition.